tests/bases.py

```diff
@@ -26,6 +26,11 @@ def setup(self):
             HTTP_AUTHORIZATION='Bearer {}'.format(self.app.config['DM_API_AUTH_TOKENS'])
         )
         self.client = self.app.test_client()
+        with self.app.app_context():
+            # Ensure there is at least one pre-approved buyer domain
+            if BuyerEmailDomain.query.filter(BuyerEmailDomain.domain_name == 'digital.gov.uk').count() == 0:
```

Comment:
Do we need this on our base class meaning it will run on every single test we run? There probably are some knock on effects that I haven't thought through but it feels like this could be something we want to avoid adding?

Reply:
It turns out that putting it as a method in `FixtureMixin` works well enough. Have added a commit to fix.